lib/tacos/dependent_slices.py

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env python3.12
 from __future__ import annotations
 
+import sys
```

Comment:
For code that isn't supposed to need `sys` (most code) I try to keep it out of the module-scope namespace, and only make it available in small scopes.

```suggestion
```